packages/web/package.json

```diff
@@ -1,33 +1,52 @@
 {
   "name": "@naturacosmeticos/natds-web",
-  "displayName": "Web Components",
+  "displayName": "NatDS React Web",
+  "private": false,
   "version": "0.6.17",
-  "description": "Natura Design System - A collection of components used by Natura Cosmeticos",
-  "main": "dist/index.js",
-  "typings": "dist/index.d.ts",
+  "description": "A collection of components for Natura Design System",
   "keywords": [
-    "design-system"
+    "design-system",
+    "natura",
+    "react",
+    "web"
   ],
+  "homepage": "https://github.com/natura-cosmeticos/natds-js/tree/master/packages/web/README.md",
+  "bugs": {
+    "url": "https://github.com/natura-cosmeticos/natds-js/issues"
+  },
+  "license": "ISC",
   "author": "Natura Cosméticos <designsystem@natura.net>",
   "contributors": [
-    "Vicente Lyrio <vicente.lyrio@vizir.com.br>",
-    "Vagner Saraiva <vagner@vizir.com.br>",
-    "Matheus Robert Lichtnow <matheus@vizir.com.br>",
     "Daniel Castro <daniel.castro@vizir.com.br>",
+    "Jimmy Andrade <jimmy.andrade@thoughtworks.com>",
+    "Matheus Robert Lichtnow <matheus@vizir.com.br>",
+    "Vagner Saraiva <vagner@vizir.com.br>",
+    "Vicente Lyrio <vicente.lyrio@vizir.com.br>",
     "Vinícius Ponciano <vinicius.ponciano@thoughtworks.com>"
   ],
-  "license": "ISC",
-  "homepage": "https://github.com/natura-cosmeticos/natds-js/tree/master/packages/web/README.md",
+  "files": [
+    "dist"
+  ],
+  "main": "dist/index.js",
```

Comment:
We must also add a browser specification here.

Reply:
✅ Done. I've included, in addition to `browser`, specific keys for `jsdelivr` and `unkpg`, to ensure that CDNs have this information as a priority.